src/Checker/ModeChecker.php

```diff
@@ -13,12 +13,15 @@ class ModeChecker extends AbstractAttributeChecker
      */
     public function create($rateMode)
     {
+        if (!is_array($rateMode)) {
```

Comment:
Why not just to `$rateMode = (array) $rateMode`, then you don't need any checks.